### ISSUE TITLE
The Computational Limits of Deep Learning 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#57]
# TL;DR
The paper suggests that Deep Learning has improved the performance of many tasks by using vast amounts of computational power, but the paper suggests that it may stall depending on the development of hardware, as the computational power required is growing larger and larger. It suggests that the financial and environmental burdens are also becoming prohibitive, so drastic improvements may be necessary.
![image](https://user-images.githubusercontent.com/24524018/91297645-a3306a00-e7d9-11ea-9ad1-e804e80bcba0.png)
![image](https://user-images.githubusercontent.com/24524018/91297650-a592c400-e7d9-11ea-8a7b-7304dc2f973d.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.05558

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

Neil C. Thompson, Kristjan Greenewald, Keeheon Lee, Gabriel F. Manso
- MIT
- MIT-IBM Watson AI Lab 
- Underwood International College 
- UnB FGA 

# Methods

# Results

# Comments
